R/derive_vars_query.R

```diff
@@ -1,6 +1,19 @@
 #' Derive Query Variables
 #'
-#' @details For each unique element in `VAR_PREFIX`, the corresponding "NAM"
+#' @details This function can be used to derive CDISC variables such as
+#'   `SMQzzNAM`, `SMQzzCD`, `SMQzzSC`, `SMQzzSCN`, and `CQzzNAM` in ADAE and
+#'   ADMH, and variables such as `SDGzzNAM`, `SDGzzCD`, and `SDGzzSC` in ADCM.
+#'   An example usage of this function can be found in the
+#'   [OCCDS vignette](https://r.roche.com/s/71c86bafe62a4200fced1/files
```

Comment:
Are these links valid?  Why do they have roche in them?

Reply:
You are right...updated to pharmaverse, do the links work now? Also there is a R CMD warning "Missing link or links in documentation object" about these objects even though the link is specified, I'm not sure which part is missing? Thanks for your help.